doc/api/index.rst

```diff
@@ -72,6 +72,7 @@ Operations on grids:
 
     grdinfo
     grdtrack
+    grdcut
```

Comment:
```suggestion
    grdcut
    grdinfo
    grdtrack
```
Make it ordered alphabetically

Reply:
Oh, and just a note that you can 'batch' up suggested changes on the 'Files Changed' tab in Github. See https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes.  Saves the CI from doing too much work.